controllers/extension-certificate-service/pkg/imagevector/imagevector.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package imagevector
+
+import (
+	"path/filepath"
+
+	"github.com/gardener/gardener-extensions/controllers/extension-certificate-service/pkg/utils"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/gardener/gardener/pkg/utils/imagevector"
+)
+
+// ImageVector is the image vector that contains al the needed images
+var ImageVector imagevector.ImageVector
```

Comment:
Please follow the same pattern as in the other controllers (not exposing the `ImageVector` var, reading the `ImageVector` via `packr`